lib/pandadoc/documents/documents.ex

```diff
@@ -210,4 +215,22 @@ defmodule Pandadoc.Documents do
     Tesla.delete(client, @documents_url <> "/#{doc_id}")
     |> Pandadoc.result()
   end
+
+  @doc """
+  Change a document's primary signer
+
+  https://developers.pandadoc.com/reference/change-signer
+  """
+  @spec change_signer(Pandadoc.client(), String.t(), String.t(), change_signer_attrs()) ::
+          Pandadoc.result()
+  def change_signer(client, document_id, recipient_id, attrs) do
```

Comment:
@seanhuggins1 if `attrs.id` is the `user_id` we could just pass in `user_id`  here
```suggestion
  def change_signer(client, document_id, recipient_id, user_id) do
```

Reply:
@seanhuggins1 wouldn't we want to update our client regardless of the API changes and we want to use that API functionality?

My main thought here about client libraries is that the API should be as clear and easy as possible for us developers to use and understand. My only struggle with the implementation was that I wasn't able to tell what was what, maybe it could be solved by adding in the fn doc or commenting next to `change_signer_attrs` that this is coming from {{documentation_link}} for clarification.

Reply:
Yeah those are fair points. I will change to `contact_id` which makes sense in the pandadoc context